src/routes/data-decode/__tests__/data-decode.e2e-spec.ts

```diff
@@ -0,0 +1,56 @@
+import * as request from 'supertest';
+import { INestApplication } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import { AppModule } from '../../../app.module';
+import { readFileSync } from 'fs';
+import { CreateDataDecodedDto } from '../entities/create-data-decoded.dto';
+import { DataDecoded } from '../../../domain/data-decoder/entities/data-decoded.entity';
+
+describe('Data decode e2e tests', () => {
+  let app: INestApplication;
+  const chainId = '5'; // Görli testnet
+
+  beforeAll(async () => {
+    const moduleRef = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleRef.createNestApplication();
+    await app.init();
+  });
+
+  it('should init the app', () => {
+    expect(app).toBeDefined();
+  });
```

Comment:
I would extract this to a different test since `AppModule` is the main application module and this test is more specific to the Data Decoder

Reply:
Makes sense, will do it 👍🏻

Reply:
I would include it in `app.module.spec.ts`. Now it might make sense to not skip the existing test in this setup

Reply:
Changed in [1d100ea921619171a818544aff2a1defbe196400](https://github.com/5afe/safe-client-gateway-nest/pull/140/commits/1d100ea921619171a818544aff2a1defbe196400)

Reply:
> I would include it in `app.module.spec.ts`. Now it might make sense to not skip the existing test in this setup

@hectorgomezv did you consider this one?

Reply:
I merged both test files, thanks!